app/controllers/register_controller.rb

```diff
@@ -1,75 +1,108 @@
 class RegisterController < ApplicationController
 
   include ApplicationHelper
-  helper_method :get_all_attribute_names_prettified, :prepare_register_name
+  helper_method :prepare_register_name, :get_description_for_register_field
```

Comment:
its not needed for prepare_register_name to share methods with the frontend unless you prefer a different way?